src/ImageSharp/ColorSpaces/CieLab.cs

```diff
@@ -127,7 +128,7 @@ public override int GetHashCode()
         }
 
         /// <inheritdoc/>
-        public override string ToString() => $"CieLab({this.L:#0.##}, {this.A:#0.##}, {this.B:#0.##})";
+        public override string ToString() => $"CieLab({FloatToString(this.L, this.A, this.B)})";
```

Comment:
inset of adding `FloatToStringUtil` (and the extra allocation that involves) you should use this syntax instead:

```
public override string ToString() => FormattableString.Invariant($"CieLab({this.L:#0.##}, {this.A:#0.##}, {this.B:#0.##})");
```

Reply:
![image](https://user-images.githubusercontent.com/1912968/46951763-2e426400-d089-11e8-9402-e26ee73770bc.png)

ColorSpaces\CieXyz.cs(97,46): error CS0103: The name 'FormattableString' does not exist in the current context [ImageSharp.csproj]

Reply:
We probably should

Reply:
Yup Windows phone is dead now and the only reason I clung on.

https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support

Reply:
I submitted PRs across the repos to increase the NETSTANDARD target to v1.3.

Reply:
Merged, and I've updated this PR from master so should be good to use now.

Reply:
Comitted with FormattableString.Invariant.